iptables.go

```diff
@@ -36,6 +36,13 @@ func newIPTables(config *bouncerConfig) (interface{}, error) {
 		CheckIptableCmds: [][]string{},
 	}
 
+	var target string
+  if strings.EqualFold(config.DenyAction, "REJECT") {
```

Comment:
i think there is an indentation problem here

Reply:
yes, should be fixed now